ga4gh/protocol.py

```diff
@@ -26,29 +58,76 @@ def convertDatetime(t):
     return int(millis)
 
 
+def getValueFromValue(value):
+    if type(value) != struct_pb2.Value:
+        raise Exception(type(value))
+    if value.WhichOneof("kind") == None:
+        raise Exception("Nothing set for %s" % value)
```

Comment:
Use string format function instead of interpolation, .e.g ``"Nothing set for {}".format(value).``. Also, is there a more specific exception class we could throw here?